pkg/popeye.go

```diff
@@ -487,14 +497,20 @@ func dumpDir() string {
 	return filepath.Join(os.TempDir(), "popeye")
 }
 
-func parseBucket(bucketURI string) (string, string, error) {
-	var bucket, key string
+func parseBucket(bucketURI string) (string, string, string, error) {
+	var host, bucket, key string
```

Comment:
A test should be provided here to make sure we're doing the right thing with minio urls